query.py

```diff
@@ -0,0 +1,394 @@
+import json
+import requests
+import networkx as nx
+
+
+class Query:
+    def __init__(
+        self,
+        service_base_url: str,
```

Comment:
I think `api_url` would be a better naming here

Reply:
Yeah but service is too generic, I think using api or fhir_api would make it clearer in Arkhn's context

Reply:
I get your point! It's changed :)